src/cloudsc_loki/CMakeLists.txt

```diff
@@ -266,6 +358,13 @@ if( HAVE_CLOUDSC_LOKI )
         DEFINITIONS ${CLOUDSC_DEFINITIONS}
     )
 
+    ecbuild_add_test(
+        TARGET dwarf-cloudsc-loki-scc-serial
+        COMMAND bin/dwarf-cloudsc-loki-scc
+        ARGS 1 1280 128
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../../..
+        OMP 1
```

Comment:
Same as with the standalone GPU version; "NVCOMPILER_ACC_CUDA_HEAPSIZE=64M" fixes it.